client/devpi/use.py

```diff
@@ -343,6 +343,18 @@ def getvenv():
         return None
     return pip.dirpath().dirpath()
 
+def current_venv():
```

Comment:
the order here could use a review, imagine devpi being made in one virtualenv and the user working in another